src/server/action_handlers/actionkit-rsvp.js

```diff
@@ -0,0 +1,9 @@
+export const displayName = () => 'ActionKit Event RSVP'
+
+export const available = () => {
+  return true
```

Comment:
I know this is a stub, but at least for prod before it's deployed this should probably be `!!process.env.ACTIONKIT_BASEURL` or something